pkg/service/role.go

```diff
@@ -264,6 +270,12 @@ func (s *roleService) Update(ctx context.Context, role *rolev3.Role) (*rolev3.Ro
 		}
 	}
 
+	//validate basic mandatory permissions that should be part of all custom roles
+	if !utils.Contains(role.Spec.Rolepermissions, partnerR) || !utils.Contains(role.Spec.Rolepermissions, organizationR) ||
+		!utils.Contains(role.Spec.Rolepermissions, projectR) {
```

Comment:
Small readability improvement
```suggestion
	if !(utils.Contains(role.Spec.Rolepermissions, partnerR) &&
		utils.Contains(role.Spec.Rolepermissions, organizationR) &&
		utils.Contains(role.Spec.Rolepermissions, projectR)) {
```